.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+    "liveServer.settings.port": 5501,
+    "eslint.debug": true,
+    "eslint.format.enable": true,
+    "editor.formatOnSave": true,
+    "[javascript]": {
+      "editor.formatOnSave": true
+    },
+    "editor.codeActionsOnSave": {
+      "source.fixAll.eslint": true
+    },
+    "eslint.alwaysShowStatus": true,
+    "files.autoSave": "onFocusChange"
```

Comment:
What does that mean?

Reply:
Tested:
- Current file is auto saved by changing to another file.
Can be deleted if this behavior is not wanted.
Description:
https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save